tests/lib/rules/no-deprecated-api.js

```diff
@@ -182,41 +182,47 @@ ruleTester.run("no-deprecated-api", rule, {
         //----------------------------------------------------------------------
         {
             code: "new (require('buffer').Buffer)()",
+            options: [{ version: "6.0.0" }],
```

Comment:
Would you remove added `version` options if it's unnecessary?

Reply:
In my opinion, the version is necessary in all the test cases. Otherwise the version declared in eslint-plugin-node's `package.json` will be used. Change the version there and you could potentially break tests.